redisdb/tox.ini

```diff
@@ -5,6 +5,8 @@ envlist =
     py{27,37}-{3.2,4.0,latest,unit}
 
 [testenv]
+description =
+    {3.2,4.0,latest}: e2e ready
```

Comment:
I would suggest `py{27,37}-latest`. We want to test py2 and py3, but the value of testing multiples versions of the system in e2e is quite low.

Reply:
IMO it still bring value to know if we ever break something from the last version that is not catched by integrations tests, why remove it then?

Reply:
Just afraid by the CI load :)

Reply:
Since we now run master nightly only, it shouldn't be an issue I think